Linter failing with these warnings:
```
- panel-graphic-walker-feedstock/recipe/meta.yaml:/requirements/host/5: host_section_needs_exact_pinnings: Linked libraries host should have exact version pinnings.
- panel-graphic-walker-feedstock/recipe/meta.yaml:/requirements/host/8: host_section_needs_exact_pinnings: Linked libraries host should have exact version pinnings.
- panel-graphic-walker-feedstock/recipe/meta.yaml:/requirements/host/7: host_section_needs_exact_pinnings: Linked libraries host should have exact version pinnings.
- panel-graphic-walker-feedstock/recipe/meta.yaml:/requirements/host/3: host_section_needs_exact_pinnings: Linked libraries host should have exact version pinnings.
```

One build failed with:
```python
DEBUG conda_anaconda_telemetry.hooks:conda_request_headers(279): Failed to collect telemetry data
Traceback (most recent call last):
  File "/opt/conda/lib/python3.12/site-packages/conda_anaconda_telemetry/hooks.py", line 277, in conda_request_headers
    yield from validate_headers(_conda_request_headers())
                                ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.12/site-packages/conda_anaconda_telemetry/hooks.py", line 211, in _conda_request_headers
    value=get_sys_info_header_value(),
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.12/site-packages/conda_anaconda_telemetry/hooks.py", line 93, in wrapper_timer
    value = func(*args, **kwargs)
            ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.12/site-packages/conda_anaconda_telemetry/hooks.py", line 151, in get_sys_info_header_value
    "conda_command": get_conda_command(),
                     ^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.12/site-packages/conda_anaconda_telemetry/hooks.py", line 124, in get_conda_command
    return context._argparse_args.cmd
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'AttrDict' object has no attribute 'cmd'
```

--- a/.gitignore
+++ b/.gitignore
@@ -6,6 +6,7 @@
 # Ignore all files and folders in root
 *
 !/conda-forge.yml
+!/abs.yaml
 
 # Don't ignore any files/folders if the parent folder is 'un-ignored'
 # This also avoids warnings when adding an already-checked file with an ignored parent.

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,0 +1,2 @@
+build_env_vars:
+  ANACONDA_ROCKET_ENABLE_PY314 : yes

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -59,5 +59,7 @@ about:
   dev_url: https://github.com/panel-extensions/panel-graphic-walker
 
 extra:
+  skip-lints:
+    - host_section_needs_exact_pinnings
   recipe-maintainers:
     - philippjfr

